frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_user.dart

```diff
@@ -86,7 +86,6 @@ class SidebarUser extends StatelessWidget {
           backgroundColor: Colors.transparent,
           child: FlowySvg(
             FlowySvgData('emoji/$iconUrl'),
-            overrideColor: false,
```

Comment:
Since there was no color provided, the color filter on the SVG widget was null anyway. We can just remove it here.